openeogeotrellis/asset_urls.py

```diff
@@ -0,0 +1,41 @@
+import logging
+from typing import Tuple
+from urllib.parse import urlparse
+
+from openeo_driver.asset_urls import AssetUrl
+from openeogeotrellis.integrations.s3proxy.exceptions import ProxyException
```

Comment:
This introduces toplevel module that adds dependencies under `openeogeotrellis.integrations`, but it's nowhere used in openeo-geopyspark-driver at the moment, so it seems that `PresignedS3AssetUrls` could also live under `openeogeotrellis.integrations.s3proxy` (or something like that?

Reply:
Will address in a separate PR and link it to https://github.com/Open-EO/openeo-geopyspark-driver/issues/1176 as well as mention you